### PR TITLE
Update B2 Buzz TVL

### PR DESCRIPTION
**Pull request by Stonepapa**
Update B2 Buzz TVL

Prior to the launch of the B2 mainnet, the TVL should be recorded on L1 networks (BNB chain, Ethereum, Polygon and others). After the launch of the B2 mainnet, the TVL should be recorded on the B2 network and should not be double counted.

We have removed the other networks in PR and have only retained the B2 network for now.

**NOTE**

> - If you would like to add a `volume` adapter please submit the PR [here](https://github.com/DefiLlama/adapters).
> - If you would like to add a `liquidations` adapter, please refer to [this readme document](https://github.com/DefiLlama/DefiLlama-Adapters/tree/main/liquidations) for details.

1. Once your adapter has been merged, it takes time to show on the UI. If more than 24 hours have passed, please let us know in Discord.
2. Please enable "Allow edits by maintainers" while putting up the PR.
3. Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
4. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
5. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
6. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
7. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---
##### Name (to be shown on DefiLlama):  
```
B² Buzz
```

##### Protocol Information: 
```
Launched by B² Network, B² Buzz is a staking event where a user can deposit major cryptocurrency assets from Bitcoin, Ethereum, BNB Chain, and Polygon. By doing so, a user will receive a certain amount of Parts which can be used to assemble Mining Rigs. Mining Rigs are employed to mine B² native tokens.
```
 
##### Twitter Link: 
```
https://twitter.com/BsquaredNetwork
```

##### List of audit links if any:
```
```

##### Website Link: 
```
https://www.bsquared.network/
```

##### Logo (High resolution, will be shown with rounded borders): 
```
https://ipfs.io/ipfs/Qmc1rHyVsnqDo4AW8e9fuhEuQsnMmaKw4Yk2KFEPgc2dCv
```

##### Current TVL: 
```
16.32 M
```

##### Treasury Addresses (if the protocol has treasury)
```
BSquared: 0x463d0908ba1F21B791883a6c27Aa82d293Dd0f69、0x9683826a04db8ae2e256e6a14b87d440c7105824
```

##### Chain:
```
BSquared
```

##### Coingecko ID (so your TVL can appear on Coingecko, leave empty if not listed): (https://api.coingecko.com/api/v3/coins/list)
```
```

##### Coinmarketcap ID (so your TVL can appear on Coinmarketcap, leave empty if not listed): (https://api.coinmarketcap.com/data-api/v3/map/all?listing_status=active,inactive,untracked&start=1&limit=10000)
```
```

##### Short Description (to be shown on DefiLlama):
```
Launched by B² Network, B² Buzz is a staking event where a user can deposit major cryptocurrency assets from Bitcoin, Ethereum, BNB Chain, and Polygon. By doing so, a user will receive a certain amount of Parts which can be used to assemble Mining Rigs. Mining Rigs are employed to mine B² native tokens.
```

##### Token address and ticker if any: 
```
```

##### Category (full list at https://defillama.com/categories) *Please choose only one: 
```
Farm
```

##### Oracle Provider(s): Specify the oracle(s) used (e.g., Chainlink, Band, API3, TWAP, etc.):
```
```
##### Implementation Details: Briefly describe how the oracle is integrated into your project:
```
```
##### Documentation/Proof: Provide links to documentation or any other resources that verify the oracle's usage:
```
```

##### forkedFrom (Does your project originate from another project):
```
```

##### methodology (what is being counted as tvl, how is tvl being calculated): 
```
Staking tokens via BSquared Network Buzz counts as TVL
```

##### Github org/user (Optional, if your code is open source, we can track activity): 
```
https://github.com/b2network
```

### CODE REVIEW

**Comment by Stonepapa** [comment]
Prior to the launch of the B2 mainnet, the TVL should be recorded on L1 networks (BNB chain, Ethereum, Polygon and others). After the launch of the B2 mainnet, the TVL should be recorded on the B2 network and should not be double counted.

We have removed the other networks in PR and have only retained the B2 network for now.

**Comment by llamatester** [comment]
The adapter at projects/bsquared exports TVL: 
        
 
 
    bsquared                  16.32 M
    
    total                    16.32 M

File: projects/bsquared/index.js [hunk 1]
```diff
@@ -1,49 +1,84 @@
...
+    ADDRESSES_BSQUARED_BSTONE,
+    ADDRESSES_BSQUARED_FDUSD,
+    ADDRESSES_BSQUARED_MATIC,
+    ADDRESSES_BSQUARED_ORDI,
+    ADDRESSES_BSQUARED_SATS,
 ];
 
-const ADDRESSES_ETHEREUM_STONE = '0x7122985656e38BDC0302Db86685bb972b145bD3C';
+const ADDRESS_UNLOCK = '0x463d0908ba1F21B791883a6c27Aa82d293Dd0f69';
+const ADDRESS_LOCK = '0x9683826a04db8ae2e256e6a14b87d440c7105824';
```
Comment (g1nt0ki) [inline projects/bsquared/index.js:36]: sorry, it is not very clear, what are these contracts? second one seems to be an EOA

Comment (Stonepapa) [reply projects/bsquared/index.js:36]: > sorry, it is not very clear, what are these contracts? second one seems to be an EOA


1) When the user is stake, the asset is transferred to the address_lock address;
2) When the user is unstake, the asset will be transferred from the address_unlock address;
3) Periodically check and transfer assets from address_lock to address_unlock;

Comment (Stonepapa) [reply projects/bsquared/index.js:36]: Do you want to provide that the monitored address is the contract address?

**Comment by Define101** [comment]
This should be for the bridge TVL (wayne dash)

**Pull request closed without merging**